# Header header
float64 time
float32 xPos
float32 yPos
float32 yaw
float32 xVel
float32 yVel
float32 zAngVel